### DISCUSSION TITLE
Request: Deploy L2 Token and support bridge RING to OP Mainet and Base, adding L2 RING to superchain token list

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#88] [Integration]
Ref:
https://github.com/ethereum-optimism/optimism-tutorial/tree/01e4f94fa2671cfed0c6c82257345f77b3b858ef/standard-bridge-standard-token

https://github.com/ethereum-optimism/ethereum-optimism.github.io?tab=readme-ov-file

@hujw77 Would you please help on this? cc @nblogist @1022ranji

**Comment by nblogist** [comment]
Hi @hujw77 Happy to help in adding RING to superchain token list :) can you please deploy the token and share the link for both the networks.

**Reply by hackfisher** [reply]
Hi @nblogist , @hujw77 is working on this, they will update here after the new xRING on Ethereum ready for use.

**Reply by nblogist** [reply]
@hujw77  pls tag me once done I'll get right on it!

**Reply by nblogist** [reply]
@hujw77 pls share the addresses when done so I can get them added to the superchain token list :)

**Comment by hackfisher** [comment]
xRING is introduced for L2 bridging, xRING can be wrapped to RING, and RING can be unwrapped to xRING through `XRINGLockBox`

https://github.com/darwinia-network/tokens?tab=readme-ov-file#deployments-on-ethereum
